src/App.jsx

```diff
@@ -6,6 +6,8 @@ function App() {
   return (
     <div className="app">
       <h1> Hello, React! </h1>
+      <h2 className="font-bold text-red-500">Is tailwind working?</h2>

```

Comment:
this way it does work, the problem was when i add reusable classes in src/css/tailwind.css, i needed to re-build